pkg/orchestration/wiring/wiringutil/knownshapes/asap_key.go

```diff
@@ -0,0 +1,43 @@
+package knownshapes
+
+import (
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+)
+
+const (
+	ASAPKeyShapeName wiringplugin.ShapeName = "voyager.atl-paas.net/ASAPKeyShape"
```

Comment:
I think we need to drop "Shape" from the end, based on what I see in the other `knownshapes`.
```
ASAPKeyShapeName wiringplugin.ShapeName = "voyager.atl-paas.net/ASAPKey"
```